packages/ui-docs-client/src/Preview/styles.js

```diff
@@ -0,0 +1,159 @@
+    checkerboard: {
+      ...checkedBackgroundBase,
+      backgroundImage: `linear-gradient(${componentTheme.gradientCheckerboard}), linear-gradient(${componentTheme.gradientCheckerboard})`
+    },
+    'checkerboard-inverse': {
+      ...checkedBackgroundBase,
+      backgroundColor: componentTheme.backgroundColorInverse,
+      backgroundImage: `linear-gradient(${componentTheme.gradientCheckerboardInverse}), linear-gradient(${componentTheme.gradientCheckerboardInverse})`
+    },
+    light: {
+      ...backgroundBase
+    },
+    inverse: {
+      backgroundColor: componentTheme.backgroundColorInverse
+    },
+    none: {
+      display: 'flex',
+      justifyContent: 'center'
+    }
+  }
+
+  const previewStyle = {
+    boxSizing: 'border-box',
+    margin: 0,
+    position: 'relative',
+    overflow: 'auto',
+
+    ...backgroundVariants[background],
+
+    ...(fullscreen && {
+      position: 'fixed',
+      width: '100vw',
+      height: '100vh'
+    }),
+
+    ...(frameless && {
+      padding: '0',
+      border: 'none',
+      margin: '1rem 0 2rem 0'
+    })
+  }
+
+  const errorStyle = {
+    background: componentTheme.backgroundError,
+    boxSizing: 'border-box',
+    margin: 0,
+    display: 'block',
+    color: componentTheme.colorError,
+    fontFamily: componentTheme.fontFamilyError,
+    fontSize: componentTheme.fontSizeError,
+    lineHeight: 1.4,
+
+    'pre, code': { color: componentTheme.colorError }
+  }
+
+  return {
+    preview: {
+      label: 'preview',
+      ...previewStyle
+    },
+
+    previewError: {
+      label: 'preview--error',
+      ...previewStyle,
+      ...errorStyle
+    },
+
+    error: {
+      label: 'preview__error',
+      ...errorStyle
+    }
+  }
+}
+
+export default generateStyle
+
+/*
+
+const converted = {
+  ".background--checkerboard, .background--checkerboard-inverse, .background--light": {
+    padding: "var(--padding)",
+    borderWidth: "var(--borderWidth)",
+    borderStyle: "solid",
+    borderColor: "var(--borderColor)",
+    borderRadius: "var(--borderRadius)",
+    backgroundColor: "var(--backgroundColorLight)"
+  },
+  ".background--checkerboard, .background--checkerboard-inverse": {
+    backgroundPosition:
+      "0 0, calc(var(--gradientCheckerboardSize) / 2) calc(var(--gradientCheckerboardSize) / 2)",
+    backgroundSize:
+      "var(--gradientCheckerboardSize) var(--gradientCheckerboardSize)"
+  },
+  ".background--checkerboard": {
+    backgroundImage:
+      "linear-gradient(var(--gradientCheckerboard)),     linear-gradient(var(--gradientCheckerboard))"
+  },
+  ".background--checkerboard-inverse": {
+    backgroundColor: "var(--backgroundColorInverse)",
+    backgroundImage:
+      "linear-gradient(var(--gradientCheckerboardInverse)),     linear-gradient(var(--gradientCheckerboardInverse))"
+  },
+  ".background--inverse": { backgroundColor: "var(--backgroundColorInverse)" },
+  ".background--none": { display: "flex", justifyContent: "center" },
+}
+*/
```

Comment:
I think you left this here by accident :)

Reply:
Ooops, sorry, thanks for finding it! :D Removed it.